sdk/storage/storage-blob/test/perfstress/track-1/upload.spec.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Aborter, BlockBlobURL } from "@azure/storage-blob";
+import { PerfStressOptionDictionary } from "@azure/test-utils-perfstress";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+import { StorageBlobTest } from "./storageTest.spec";
+dotenv.config({ path: "../../../.env" });
+
+interface StorageBlobUploadTestOptions {
+  size: number;
+}
+
+export class StorageBlobUploadTest extends StorageBlobTest<StorageBlobUploadTestOptions> {
+  public options: PerfStressOptionDictionary<StorageBlobUploadTestOptions> = {
+    size: {
+      required: true,
+      description: "Size in bytes",
+      shortName: "sz",
+      longName: "size",
+      defaultValue: 10
+    }
+  };
+
+  async runAsync(): Promise<void> {
+    const blockBlobClient = BlockBlobURL.fromContainerURL(
```

Comment:
Each instance should upload to same blob.

Reply:
Done